chat_bot/prompts.py

```diff
@@ -7,22 +7,22 @@
+        "climate change": "This is a conversation between two friends about climate change.\n\nFriend_q: Have you heard about climate change?\nYou: Yeah, I've been hearing about it a lot lately.\nFriend_q: Did you hear that it causes all sorts of problems like sea level rise, tropical storms and freak weather?\nYou: It's definitely something to be concerned about. It's only going to get worse in the future. What can we do to address it?\nFriend_a: We can reduce our fossil fuel consumption and use more renewable energy sources.\nYou: Really? What are some examples?\nFriend_a: There are a lot of different ways to reduce fossil fuel consumption. You can drive less, use less energy in your home, and switch to renewable energy sources like solar and wind power.",
+        "deforestation": "This is a conversation between two friends about deforestation.\n\nFriend_q: What are your thoughts on deforestation?\nYou: I think it's a huge problem. Deforestation is responsible for about 20% of global greenhouse gas emissions, so it's a major contributor to climate change. What do you think?\nFriend_a: It's also responsible for the loss of habitat for millions of animals, and it's a major contributor to global warming.\nYou: What can we do to help the situation?\nFriend_a: I'm not sure. I think we need to be more conscious about the things we buy and how they're made. We also need to demand that companies use sustainable practices.\nYou: I agree.\nFriend_q: Do you try to save paper whenever you can?\nYou: Yes, I do try to save paper whenever I can. For example, I print documents on both sides of the paper whenever possible.",
+        "extinction" : "This is a conversation between two friends about endangered species.\n\nFriend_q: Did you hear that the dinosaurs went extinct?\nYou: I know, it's so sad. They were such cool animals.\nFriend_q: It's really sad that so many animals are in danger of becoming extinct. What can we do to help protect them?\nYou: We can protect the environment, and conserve forests and land, to conserve their habitats. What animals are in danger?\nFriend_a: There are so many animals that are in danger of becoming extinct. Some of the animals that are in the most danger are the Tigers, Pandas, and the Gorillas.\nYou: That's so sad. How can we help?\nFriend_a: We can create awareness about the importance of these animals, and the danger they are in."
```

Comment:
I too am sad that dinosaurs went extinct... :(

Reply:
@anoushkamazumdar this was so funny